pkg/daemon/ceph/agent/flexvolume/controller.go

```diff
@@ -335,7 +358,9 @@ func (c *Controller) GetGlobalMountPath(input GlobalMountPathInput, globalMountP
 }
 
 // GetClientAccessInfo obtains the cluster monitor endpoints, username and secret
-func (c *Controller) GetClientAccessInfo(clusterNamespace string, clientAccessInfo *ClientAccessInfo) error {
+func (c *Controller) GetClientAccessInfo(args []string, clientAccessInfo *ClientAccessInfo) error {
```

Comment:
instead of a `[]string` could we either pass individual args or possibly add them to the `ClientAccessInfo` type? this way we could keep the params strongly typed.